packages/odyssey-react-mui/src/labs/SideNav.tsx

```diff
@@ -0,0 +1,572 @@
+  "& > a": {
+    color: `${odysseyDesignTokens.TypographyColorHeading} !important`,
+    pointerEvents: isDisabled ? "none" : "auto",
+  },
+  "& > a:hover": {
+    textDecoration: "none",
+    cursor: isDisabled ? "default" : "pointer",
+  },
+  "& > a:visited": {
+    color: odysseyDesignTokens.TypographyColorHeading,
+    fontSize: odysseyDesignTokens.TypographyScale0,
+  },
+}));
+
+const SideNavListItemContainer = styled("li", {
+  shouldForwardProp: (prop) =>
+    prop !== "odysseyDesignTokens" &&
+    prop !== "isSelected" &&
+    prop !== "isDisabled",
+})<{
+  odysseyDesignTokens: DesignTokens;
+  isSelected?: boolean;
+  disabled?: boolean;
+  isDisabled?: boolean;
+}>(({ odysseyDesignTokens, isSelected, isDisabled }) => ({
+  display: "flex",
+  alignItems: "center",
+  minHeight: "48px",
+  opacity: isDisabled ? "0.38" : "1",
+  cursor: isDisabled ? "default" : "pointer",
+  pointerEvents: isDisabled ? "none" : "auto",
+  backgroundColor: isSelected ? odysseyDesignTokens.HueNeutral50 : "auto",
+  "&:hover": {
+    backgroundColor: !isDisabled ? odysseyDesignTokens.HueNeutral50 : "auto",
+  },
+  "&:last-child": {
+    marginBottom: odysseyDesignTokens.Spacing2,
+  },
+  "& > a": {
+    display: "flex",
+    alignItems: "center",
+    width: "100%",
+    paddingRight: odysseyDesignTokens.Spacing4,
+    paddingLeft: odysseyDesignTokens.Spacing4,
+    color: `${odysseyDesignTokens.TypographyColorHeading} !important`,
+    pointerEvents: isDisabled ? "none" : "auto",
+  },
+  "& > a:hover": {
+    textDecoration: "none",
+    cursor: isDisabled ? "default" : "pointer",
+  },
+  "& > a:visited": {
+    color: odysseyDesignTokens.TypographyColorHeading,
+    fontSize: odysseyDesignTokens.TypographyScale0,
+  },
+}));
+
+const SideNavFooterContainer = styled("div", {
+  shouldForwardProp: (prop) => prop !== "odysseyDesignTokens",
+})(({ odysseyDesignTokens }: { odysseyDesignTokens: DesignTokens }) => ({
+  paddingTop: odysseyDesignTokens.Spacing2,
+  paddingBottom: odysseyDesignTokens.Spacing2,
+  display: "flex",
+  justifyContent: "center",
+  flexWrap: "wrap",
+  alignItems: "center",
+  fontSize: odysseyDesignTokens.TypographySizeOverline,
+  "& > a": {
+    color: `${odysseyDesignTokens.TypographyColorHeading} !important`,
+  },
+  "& > a:hover": {
+    textDecoration: "none",
+  },
+  "& > a:visited": {
+    color: odysseyDesignTokens.TypographyColorHeading,
+  },
+  "& > a:not(:last-child)::after": {
+    marginLeft: odysseyDesignTokens.Spacing4,
+    marginRight: odysseyDesignTokens.Spacing4,
+    color: odysseyDesignTokens.HueNeutral300,
+    content: '" | "',
+  },
+}));
+
+const SideNavFooter = ({ id, label, href }: SideNavFooterItem) => {
+  return (
+    <Link key={id} href={href}>
+      {label}
+    </Link>
+  );
+};
+
+const SideNavItemContent = ({
+  id,
+  label,
+  href,
+  target,
+  startIcon,
+  severity,
+  statusLabel,
+  endIcon,
+  onClick,
+  isSelected,
+  isDisabled,
+}: Pick<
```

Comment:
We haven't been doing this `Pick` technique for types in Odyssey. @KevinGhadyani-Okta , is this something we should consider adopting?

Reply:
@jordankoschei-okta yeah, this was a suggestion from Kevin.

Reply:
We use `Pick` in a few places. This is fine by me